bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/Ballot/BallotValidationExtensions.cs

```diff
@@ -0,0 +1,200 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ElectionGuard.Encryption.Ballot
+{
+    public class BallotValidationResult
```

Comment:
this is also one of the fun parts. when we check validity it will blow up in a few cases early, but in most casesd it will check both the data sanitization and the encryption values and give you back a recursive collection of results that say what failed and why.